chain/src/consume_unverified.rs

```diff
@@ -1,414 +1,261 @@
+                        }
 
-        // Mainly for test: give an empty thread_name
-        let mut thread_builder = thread::Builder::new();
-        if let Some(name) = thread_name {
-            thread_builder = thread_builder.name(name.to_string());
-        }
-        let tx_control = self.shared.tx_pool_controller().clone();
-
-        let chain_jh = thread_builder
-            .spawn(move || loop {
-                select! {
-                    recv(process_block_receiver) -> msg => match msg {
-                        Ok(Request { responder, arguments: (block, verify) }) => {
-                            let instant = Instant::now();
-
-                            let _ = tx_control.suspend_chunk_process();
-                            let _ = responder.send(self.process_block(block, verify));
-                            let _ = tx_control.continue_chunk_process();
-
-                            if let Some(metrics) = ckb_metrics::handle() {
-                                metrics
-                                    .ckb_block_process_duration
-                                    .observe(instant.elapsed().as_secs_f64());
-                            }
-                        },
-                        _ => {
-                            error!("process_block_receiver closed");
-                            break;
-                        },
+                        let _ = self.tx_pool_controller.continue_chunk_process();
                     },
-                    recv(truncate_receiver) -> msg => match msg {
-                        Ok(Request { responder, arguments: target_tip_hash }) => {
-                            let _ = tx_control.suspend_chunk_process();
-                            let _ = responder.send(self.truncate(&target_tip_hash));
-                            let _ = tx_control.continue_chunk_process();
-                        },
-                        _ => {
-                            error!("truncate_receiver closed");
-                            break;
-                        },
+                    Err(err) => {
+                        error!("unverified_block_rx err: {}", err);
+                        return;
                     },
-                    recv(signal_receiver) -> _ => {
-                        info!("ChainService received exit signal, exit now");
-                        break;
-                    }
+                },
+                recv(self.truncate_block_rx) -> msg => match msg {
+                    Ok(Request { responder, arguments: target_tip_hash }) => {
+                        let _ = self.tx_pool_controller.suspend_chunk_process();
+                        let _ = responder.send(self.processor.truncate(&target_tip_hash));
+                        let _ = self.tx_pool_controller.continue_chunk_process();
+                    },
+                    Err(err) => {
+                        info!("truncate_block_tx has been closed,err: {}", err);
```

Comment:
add a space here "closed,err:"
```suggestion
                        info!("truncate_block_tx has been closed, err: {}", err);
```